client/js/components/button/BackToTopButton.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+  <dp-button
+    :aria-label="Translator.trans('backToTop')"
+    :title="Translator.trans('backToTop')"
+    variant="outline"
+    color="primary"
```

Comment:
Does not have to be set, as color defaults to primary.

Reply:
755482495d4967cdd66fd9a3a1f8fd5770c41d62